library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/optionselectormocks/DropDownListView.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package android.widget
+
+import android.content.Context
+import android.view.ViewGroup
+
+internal class DropDownListView(context: Context) : ViewGroup(context) {
```

Comment:
do we need this class? `android.widget.DropDownListView` will always be in the classpath, because it is a system API.

Reply:
yeah the main issue is that all these classes are either internal to their packages or `hidden` so you cannot use them in your code. I even tried to look for them in the `ClassLoader` at runtime and you can't find them.